src/store/item.ts

```diff
@@ -30,6 +30,11 @@ export const useItemStore = defineStore("item", {
       return this.saveData(podcastname, docname)
     },
 
+    async updatesoltItem(state: State,item: any, podcastname: string, docname: string) {
```

Comment:
`updatesoltItem`
not sure what this is meant to mean?

Reply:
After reordering items then we need to update state value so I have created this function (`updatesoltItem`)

Reply:
understood, i'm just not sure what you meant to write, presumably `update slot Item` but misspelled `slot` dnd didn't capitalise it.
If thats right, suggest renaming it to `updateSlotItem`